cns/client_test.go

```diff
@@ -720,6 +720,48 @@ func TestClient(t *testing.T) {
 		fileBackingInfo := queryResult.Volumes[0].BackingObjectDetails.(*cnstypes.CnsVsanFileShareBackingDetails)
 		t.Logf("File Share Name: %s with accessPoints: %+v", fileBackingInfo.Name, fileBackingInfo.AccessPoints)
 
+		// Test Configure ACLs
+		netPerms := make([]vsanfstypes.VsanFileShareNetPermission, 0)
+		netPerms = append(netPerms, vsanfstypes.VsanFileShareNetPermission{
+			Ips:         "192.168.124.2",
+			Permissions: "READ_ONLY",
+		})
+
+		var permission []vsanfstypes.VsanFileShareNetPermission
```

Comment:
Looks like this `permission` var is unused?

Reply:
Removed